uk-ua/java-ua.html.markdown

```diff
@@ -0,0 +1,782 @@
+---
+language: java
+contributors:
+    - ["Jake Prather", "http://github.com/JakeHP"]
+    - ["Jakukyo Friel", "http://weakish.github.io"]
+    - ["Madison Dickson", "http://github.com/mix3d"]
+    - ["Simon Morgan", "http://sjm.io/"]
+    - ["Zachary Ferguson", "http://github.com/zfergus2"]
+    - ["Cameron Schermerhorn", "http://github.com/cschermerhorn"]
+    - ["Rachel Stiyer", "https://github.com/rstiyer"]
+translators:
+    - ["Oleksandr Tatarchuk", "https://github.com/tatarchuk"]
+filename: LearnJava.java
```

Comment:
To the filename (before the extension), that's why I added a comment on that line